R/connect.R

```diff
@@ -818,6 +818,33 @@ Connect <- R6::R6Class(
       self$GET(path, query = query)
     },
 
+    #' @description Get content usage data.
+    #' @param from Optional `Date` or `POSIXt`; start of the time window. If a
+    #' `Date`, coerced to `YYYY-MM-DDT00:00:00` in the caller's time zone.
+    #' @param to Optional `Date` or `POSIXt`; end of the time window. If a
+    #' `Date`, coerced to `YYYY-MM-DDT23:59:59` in the caller's time zone.
+    inst_content_hits = function(from = NULL, to = NULL) {
```

Comment:
I don't think this method name benefits from having the `inst_` prefix on it. 

More of a philosophical musing, but I also am not sure it's worth packing more things into methods on the `Connect` R6 object. `get_usage()` is a pretty thin wrapper around this. We could just put this inside of `get_usage()`, not sure there's much benefit to separating things like this. I know this is a pattern in parts of the package, but that doesn't mean we have to keep doing it.

Reply:
Agreed that I don't think the method name at all benefits from `inst_` prefix; to the reader, it isn't at all clear what it means.

I would also be completely fine with beginning to move everything into the functions; I agree that I don't think we get much benefit at all from having the logic in the R6 object. In fact, I think it _adds_ confusion: it was hard for me to decide where we want some of the cutoff to be.